core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -374,40 +374,4 @@ object LogConfig {
     MessageTimestampDifferenceMaxMsProp -> KafkaConfig.LogMessageTimestampDifferenceMaxMsProp,
     MessageDownConversionEnableProp -> KafkaConfig.LogMessageDownConversionEnableProp
   )
-
-
-  /**
-   * Copy the subset of properties that are relevant to Logs. The individual properties
```

Comment:
We moved this here from `KafkaServer`. Not sure why we want to revert that. `LogConfig` seems like the right place to have a function to extract the log config properties.